src/main/java/it/pagopa/pn/delayer/PaperDeliveryJobRunner.java

```diff
@@ -0,0 +1,34 @@
+package it.pagopa.pn.delayer;
+
+import it.pagopa.pn.delayer.service.HighPriorityBatchService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class PaperDeliveryJobRunner implements CommandLineRunner {
+
+    private final HighPriorityBatchService highPriorityBatchService;
+    private final ApplicationContext applicationContext;
+
+    @Override
+    public void run(String... args) {
+        if(args.length != 0 && StringUtils.isNotBlank(args[0])){
+            log.info("Starting batch for pk: {}", args[0]);
+            highPriorityBatchService.initHighPriorityBatch(args[0], new HashMap<>()).block();
```

Comment:
nel metodo `highPriorityBatchService.initHighPriorityBatch(args[0], new HashMap<>())`, passerei anche `var startExcecutionBatch = Instant.now`, in modo tale che tutti i chuck abbiano lo stesso instant.now e così non siamo affetti da eventuali casi limite (passeremo questo valore in  `Instant deliveryWeek = paperDeliveryUtils.calculateNextWeek(startExcecutionBatch);`).